dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -9534,6 +9596,7 @@ CONFIG_PAGE_EXTENSION=y
 # CONFIG_DEBUG_PAGEALLOC is not set
 CONFIG_SLUB_DEBUG=y
 # CONFIG_SLUB_DEBUG_ON is not set
+# CONFIG_SLUB_RCU_DEBUG is not set
```

Comment:
May this be useful for us?

Reply:
https://lore.kernel.org/all/20240809-kasan-tsbrcu-v8-2-aef4593f9532@google.com/

The comments say

```
+	  This might degrade performance.
+	  Unfortunately this also prevents a very specific bug pattern from
+	  triggering (insufficient checks against an object being recycled
+	  within the RCU grace period); so this option can be turned off even on
+	  KASAN builds, in case you want to test for such a bug.
+
+	  If you're using this for testing bugs / fuzzing and care about
+	  catching all the bugs WAY more than performance, you might want to
+	  also turn on CONFIG_RCU_STRICT_GRACE_PERIOD.
+
```

Reply:
Yes, please enable `CONFIG_SLUB_RCU_DEBUG`, at least on most instances. Is is theoretically possible to have bugs that will be hidden by this flag (though I'm not sure if I've ever seen a single instance of such a bug); but for significantly more bugs, this flag will make it possible for KASAN to notice UAFs.

Reply:
Of particular interest is probably that `CONFIG_SLUB_RCU_DEBUG` is nowadays basically necessary to directly detect UAF of `struct file`.